docs/submission/submitting-molecular-data2.md

```diff
@@ -0,0 +1,439 @@
+---
+id: submitting-molecular-data2
+title: Preparing and Validating Molecular Data2
+platform_key: DOCS_VALIDATING_MOLECULAR_DATA
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The ARGO DCC requires that necessary information describing molecular data files, called metadata, is be submitted at the same time as the molecular data to facilitate automated downstream analysis. This metadata is submitted to the ARGO metadata repository, called Song. 
+
+## Preparing Molecular Metadata Payloads
+
+### Understanding the payload format
+
+Molecular data is submitted in conjunction with metadata submitted in JSON format (called a payload), which is validated against JSON Schema to ensure data quality. The first step of submitting molecular data is to prepare the metadata payloads conforming to the most recent JSON schema that has been defined by the DCC. Once the metadata has been prepared, both the payload and the molecular data files can be [submitted](/docs/submission/submitting-molecular-data) together. 
```

Comment:
suggested changes:

Molecular data is submitted in conjunction with metadata submitted in JSON format, called a payload. The payload is validated against a JSON Schema to ensure data quality. The first step of submitting molecular data is to prepare the metadata payloads conforming to the most recent JSON schema that has been defined by the DCC. Once the metadata has been prepared, both the payload and the molecular data files can be [submitted](/docs/submission/submitting-molecular-data) together.

Reply:
done